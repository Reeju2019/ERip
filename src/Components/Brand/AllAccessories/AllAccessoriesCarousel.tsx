import * as React from 'react'
import { Carousel, Container, Image } from 'react-bootstrap'

const AllAccessories: React.FunctionComponent = () => {
  return (
    <>
      <Container fluid className='carousel-container-1 mt-2' style={{ backgroundColor: 'blue' }}>
        <div className='row carousel-content mx-4'>
          <div id='carouselExampleIndicators' className='carousel slide' data-bs-ride='true'>
            <Carousel>
              <Carousel.Item>
                <div className='d-flex flex-row'>
                  <div className='w-100 d-flex justify-content-center align-item-center align-self-center'>
                    <h1 className='txt col-8 justify-content-start text-white fs-1'>
                      All Original Accessories
                    </h1>
                  </div>
                  <Image
                    style={{ width: '16vw' }}
                    className='d-flex justify-content-center'
                    src='/static/Mobile_Services/usb-cable.png'
                    alt='First slide'
                  />
                </div>
              </Carousel.Item>
              <Carousel.Item>
                <div className='d-flex flex-row'>
                  <div className='w-100 d-flex justify-content-center align-item-center align-self-center'>
                    <h1 className='txt col-8 justify-content-start text-white fs-1'>
                      All Original Accessories
                    </h1>
                  </div>
                  <Image
                    style={{ width: '16vw' }}
                    className='d-flex  justify-content-center'
                    src='/static/Mobile_Services/usb-cable.png'
                    alt='First slide'
                  />
                </div>
              </Carousel.Item>
              <Carousel.Item>
                <div className='d-flex flex-row'>
                  <div className='w-100 d-flex justify-content-center align-item-center align-self-center'>
                    <h1 className='txt col-8 justify-content-start text-white fs-1'>
                      All Original Accessories
                    </h1>
                  </div>
                  <Image
                    style={{ width: '16vw' }}
                    className='d-flex justify-content-center'
                    src='/static/Mobile_Services/usb-cable.png'
                    alt='First slide'
                  />
                </div>
              </Carousel.Item>
            </Carousel>
          </div>
        </div>
      </Container>
    </>
  )
}

export default AllAccessories
