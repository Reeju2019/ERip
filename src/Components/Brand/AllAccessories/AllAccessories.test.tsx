import React from 'react'
import { render, screen } from '@testing-library/react'
import AllAccessories from './AllAccessoriesCarousel'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'

test('all accessories', () => {
  render(<AllAccessories />)
})

describe('App rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<AllAccessories />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
<<<<<<< HEAD
    
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<AllAccessories />)
    ReactDOM.unmountComponentAtNode(div)
=======
>>>>>>> cb5880a93c98ef130e6353a40a30f7c7065eaaff
  })
})
