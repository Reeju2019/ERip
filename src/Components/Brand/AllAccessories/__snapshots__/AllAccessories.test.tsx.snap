// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<div
  className="carousel-container-1 mt-2 container-fluid"
  style={
    Object {
      "backgroundColor": "blue",
    }
  }
>
  <div
    className="row carousel-content mx-4"
  >
    <div
      className="carousel slide"
      data-bs-ride="true"
      id="carouselExampleIndicators"
    >
      <div
        className="carousel slide"
        onKeyDown={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
      >
        <div
          className="carousel-indicators"
        >
          <button
            aria-current={true}
            aria-label="Slide 1"
            className="active"
            data-bs-target=""
            onClick={[Function]}
            type="button"
          />
          <button
            aria-current={false}
            aria-label="Slide 2"
            data-bs-target=""
            onClick={[Function]}
            type="button"
          />
          <button
            aria-current={false}
            aria-label="Slide 3"
            data-bs-target=""
            onClick={[Function]}
            type="button"
          />
        </div>
        <div
          className="carousel-inner"
        >
          <div
            className="active carousel-item"
          >
            <div
              className="d-flex flex-row"
            >
              <div
                className="w-100 d-flex justify-content-center align-item-center align-self-center"
              >
                <h1
                  className="txt col-8 justify-content-start text-white fs-1"
                >
                  All Original Accessories
                </h1>
              </div>
              <img
                alt="First slide"
                className="d-flex justify-content-center"
                src="/static/Mobile_Services/usb-cable.png"
                style={
                  Object {
                    "width": "16vw",
                  }
                }
              />
            </div>
          </div>
          <div
            className="carousel-item"
          >
            <div
              className="d-flex flex-row"
            >
              <div
                className="w-100 d-flex justify-content-center align-item-center align-self-center"
              >
                <h1
                  className="txt col-8 justify-content-start text-white fs-1"
                >
                  All Original Accessories
                </h1>
              </div>
              <img
                alt="First slide"
                className="d-flex  justify-content-center"
                src="/static/Mobile_Services/usb-cable.png"
                style={
                  Object {
                    "width": "16vw",
                  }
                }
              />
            </div>
          </div>
          <div
            className="carousel-item"
          >
            <div
              className="d-flex flex-row"
            >
              <div
                className="w-100 d-flex justify-content-center align-item-center align-self-center"
              >
                <h1
                  className="txt col-8 justify-content-start text-white fs-1"
                >
                  All Original Accessories
                </h1>
              </div>
              <img
                alt="First slide"
                className="d-flex justify-content-center"
                src="/static/Mobile_Services/usb-cable.png"
                style={
                  Object {
                    "width": "16vw",
                  }
                }
              />
            </div>
          </div>
        </div>
        <a
          className="carousel-control-prev"
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            aria-hidden="true"
            className="carousel-control-prev-icon"
          />
          <span
            className="visually-hidden"
          >
            Previous
          </span>
        </a>
        <a
          className="carousel-control-next"
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            aria-hidden="true"
            className="carousel-control-next-icon"
          />
          <span
            className="visually-hidden"
          >
            Next
          </span>
        </a>
      </div>
    </div>
  </div>
</div>
`;
